components/feature/prompts/build.gradle

```diff
@@ -34,6 +34,7 @@ dependencies {
     implementation project(':support-ktx')
     implementation project(':support-utils')
     implementation project(':ui-icons')
+    implementation project(':feature-loginsexceptions')
```

Comment:
Q - is there a way to not make the downstream consumers of prompt-feature also _have_ to add this feature since it's optional?

Reply:
I can see two options:
* Use `compileOnly`. That works if no class of that is actually somehow needed at runtime if you do not use that feature. But it's rather awkward since if you do it will randomly fail and no tooling will warn you about it.
* For everything that `feature-prompts` needs access to, create an interface in `feature-prompts`. Then in your new component implement those interfaces. That way `feature-prompts` does not need to know about the implementation and you are essentially reversing the dependencies. This will need some setup/glue code so that you can tell `feature-prompts` about which implementation to use though.